_assets/borg.css

```diff
@@ -21,6 +21,45 @@ header h1 {
 	text-align: center;
 }
 
+nav {
+	display: flex;
+	justify-content: center;
+}
+
+nav > ol {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	gap: 1rem;
+	padding: 0;
+}
+
+nav > ol > li {
+	list-style: none;
+}
+
+/* mono system font stack courtesy of https://qwtel.com/posts/software/the-monospaced-system-ui-css-font-stack/ */
+nav > ol > li > a {
+	font-size: 1.1rem;
```

Comment:
is 1.0 too small?

i imagine that when scaling, the rendering quality depends a bit on the scaling factor and especially "uneven" factors like 1.1 might be worse quality than e.g. 1.0?

Reply:
Hmm, good question. I thought that because pretty much all fonts are vectorized now, arbitrary scaling was not an issue. (I never worried about using, e.g. 19px before)

Reply:
Ah, so they are rendered from vectors for the requested size? OK, if that is the case, guess this is not an issue.